### PR TITLE
Add makefile and include building instructions in README

### PR DESCRIPTION
**Pull request by cartesiancat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ A graphical editor for commutative diagrams that exports to tikz-cd.
 - A history system with undo and redo.
 - Panning.
 
+## Building
+1. Clone the repository.
+1. `git submodule update --init --recursive`.
+1. `yarn`.
```
Comment (varkor) [inline README.md]: Isn't it necessary to run these commands from `src/KaTeX`?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.ONESHELL:
+all:
```
Comment (varkor) [inline makefile:3]: There should probably be a `.PHONY: all` above.

File: README.md [hunk 3]
```diff
@@ -22,6 +22,23 @@ A graphical editor for commutative diagrams that exports to tikz-cd.
...
 - Panning.
 
+## Building
+Clone the repository. Run:
+```
+make
+```
+Open `src/index.html` in your favourite web browser.
+
+If make doesn't work try
```
Comment (varkor) [inline README.md]: Let's skip this (assuming anyone contributing is likely to know how `make` works) and just have the suggestion to open an issue if it doesn't work.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,23 @@ A graphical editor for commutative diagrams that exports to tikz-cd.
...
+```
+git submodule update --init --recursive
+cd src/KaTeX
+yarn
+yarn build
+```
+
+If that doesn't work make an issue
```
Comment (varkor) [inline README.md]:
```suggestion
If that doesn't work, [open an issue](https://github.com/varkor/quiver/issues/new) detailing the problem.
```

**Comment by varkor** [comment]
Looks good after the comments are addressed.

File: README.md [hunk 5]
```diff
@@ -22,6 +22,26 @@ A graphical editor for commutative diagrams that exports to tikz-cd.
...
+Clone the repository. Run:
+```
+make
+```
+Open `src/index.html` in your favourite web browser.
+
+If that doesn't work, [open an
+issue](https://github.com/varkor/quiver/issues/new) detailing the problem.
+
+<blockquote>
```
Comment (varkor) [inline README.md:37]: How common a problem is this?

Comment (varkor) [reply README.md:37]: This appears to be a very specific problem with an alternative `yarn` command on Unix. I would be inclined to remove the `<blockquote>`, assuming that users are able to search for solutions to this error themselves.

Comment (cartesiancat) [reply README.md:37]: It affects Debian stable, Ubuntu (except the latest version), and derivatives of either (e.g. Linux Mint). So most distros that use `apt` for package management.

**Comment by varkor** [comment]
Thanks for these changes! Sorry for the delay — quiver hasn't been my priority lately.

**Pull request merged**